### ISSUE TITLE
bug(core): events with the same name but different attributes will panic

### ISSUE THREAD

**Issue by neverDefined** [berachain/polaris#550]
Cosmos emits events that have the same name but different attributes:

```go
ctx.EventManager().EmitEvent(
			sdk.NewEvent(govtypes.EventTypeSubmitProposal,
				sdk.NewAttribute(govtypes.AttributeKeyVotingPeriodStart, fmt.Sprintf("%d", proposal.Id)),
			),
		)
		
	ctx.EventManager().EmitEvent(
		sdk.NewEvent(
			types.EventTypeSubmitProposal,
			sdk.NewAttribute(types.AttributeKeyProposalID, fmt.Sprintf("%d", proposalID)),
			sdk.NewAttribute(types.AttributeKeyProposalMessages, msgsStr),
		),
	)

```

this case is not handled and will cause the chain to panic.

**Comment by neverDefined** [comment]
example :

https://vscode.dev/github/dapphub/ds-test2.0.20230405143654-e053c5207b76/x/gov/keeper/msg_server.go#L73

https://vscode.dev/github/dapphub/ds-test2.0.20230405143654-e053c5207b76/x/gov/keeper/proposal.go#L102

**Comment by calbera** [comment]
temp fix #558 ignore all attributes on `SubmitProposal` event
